db/migrate/20221031115646_create_notes.rb

```diff
@@ -0,0 +1,16 @@
+class CreateNotes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :notes do |t|
+      t.boolean :issue, default: false
```

Comment:
For future proofing, it might be better to have an enum for the type, so easier to add new types later?